spec/hashie/parsers/yaml_erb_parser_spec.rb

```diff
@@ -1,6 +1,21 @@
 require 'spec_helper'
 
 describe Hashie::Extensions::Parsers::YamlErbParser do
+  describe '#perform' do
```

Comment:
There's already a `describe '.perform'` below, so combine them. Something like this:

```ruby
describe '.perform' do
   context 'a file' do
   end
   context 'Pathname' do
   end
end
```